pkgs/development/compilers/rust/clippy.nix

```diff
@@ -11,7 +11,7 @@ rustPlatform.buildRustPackage {
   # changes hash of vendor directory otherwise
   dontUpdateAutotoolsGnuConfigScripts = true;
 
-  buildInputs = [ rustc ] ++ stdenv.lib.optionals stdenv.isDarwin [ Security ];
+  buildInputs = [ rustc llvm_9 ] ++ stdenv.lib.optionals stdenv.isDarwin [ Security ];
```

Comment:
Please add `passthru.llvm = llvmShared;` in `pkgs/development/compilers/rust/rustc.nix`
and than replace llvm_9 with `rustc.llvm`. That way we always keep llvm of clippy in sync with what rustc uses.

Reply:
What a neat trick! I've done so.